recipes/taglib.recipe

```diff
@@ -30,6 +30,11 @@ class Recipe(recipe.Recipe):
         if self.config.target_platform == Platform.ANDROID:
             # configure for android
             self.configure_options += ' -DANDROID_NDK=1 '
+        if self.config.target_platform == Platform.LINUX:
```

Comment:
This should be done only for Trusty, other distros might install libGL in a different path. Also link to the bug in case someone needs to understand the hack